### PR TITLE
feat: add flags filters for feed config

### PR DESCRIPTION
**Pull request by capJavert**
- apply new flags to filters
- send new filters in config generator
- move customFeed query to feed endpoint and custom feed config name (confirmed name with Vas)
- currently not moving to RR because I don't have time to test if any edge cases, will log

### CODE REVIEW

File: __tests__/feeds.ts [hunk 1]
```diff
@@ -4348,13 +4425,15 @@ describe('query customFeed', () => {
...
+      .post('/feed.json', {
         user_id: '1',
         page_size: 10,
         offset: 0,
         total_pages: 1,
         fresh_page_size: '4',
         allowed_tags: ['webdev', 'html', 'data'],
+        feed_config_name: FeedConfigName.CustomFeedV1,
+        disable_engagement_filter: false,
       })
```
Comment (capJavert) [inline __tests__/feeds.ts:4437]: Apply config name and filters

File: src/integrations/feed/types.ts [hunk 2]
```diff
@@ -108,3 +110,15 @@ export const baseFeedConfig: Partial<FeedConfig> = {
...
+  order_by?: FeedFlags['orderBy'];
+  disable_engagement_filter?: FeedFlags['disableEngagementFilter'];
+  thresholds?: {
+    min_thresholds: {
+      upvotes?: FeedFlags['minUpvotes'];
+      views?: FeedFlags['minViews'];
+    };
+  };
+  min_day_range?: FeedFlags['minDayRange'];
+};
```
Comment (capJavert) [inline src/integrations/feed/types.ts:124]: This is what we support on API side so far

File: src/schema/feeds.ts [hunk 3]
```diff
@@ -233,7 +233,7 @@ export const typeDefs = /* GraphQL */ `
     """
     Minimum day range
     """
-    maxDayRange: Int
+    minDayRange: Int
```
Comment (capJavert) [inline src/schema/feeds.ts:236]: was just typo, I mixed up how its called

File: src/common/feedGenerator.ts [hunk 4]
```diff
@@ -307,6 +317,47 @@ const sourcesToFilters = ({
...
+  // we set flags only for custom feeds
+  if (feed.type !== FeedType.Custom) {
+    return {};
+  }
+
+  if (feed.flags.orderBy) {
+    flagFilters.order_by = feed.flags.orderBy;
+  }
+
+  flagFilters.disable_engagement_filter = !!feed.flags.disableEngagementFilter;
```
Comment (capJavert) [inline src/common/feedGenerator.ts:341]: by default `false` (as in feed side) when flag is not present in jsonb

File: src/schema/feeds.ts (hunk 3 above)
Comment (omBratteng) [reply src/schema/feeds.ts:236]: :lolsob:

**Review by idoshamun — APPROVED** [review]

**Review by rebelchris — APPROVED** [review]
Awesome

File: src/common/feedGenerator.ts [hunk 5]
```diff
@@ -161,6 +164,13 @@ const getRawFiltersData = async (
         .from(SourceMember, 't')
         .where('"userId" = $2'),
     ),
+    rawFilterSelect(con, 'feeds', (qb) =>
```
[the remark this answers is not in the document]
Comment (capJavert) [reply src/common/feedGenerator.ts:167]: ah yeah, we can do that, if feedid equal userid

[the remark this answers is not in the document]
Comment (capJavert) [reply src/common/feedGenerator.ts]: Added in 9f54b9bcaf554749a65253cc885dd2cb69d8d458

**Review by rebelchris — APPROVED** [review]
